gammapy/data/observations.py

```diff
@@ -172,15 +186,28 @@ def create(
         obs : `gammapy.data.MemoryObservation`
         """
         if tstart is None:
-            tstart = Quantity(0.0, "hr")
+            tstart = reference_time.copy()
 
         if tstop is None:
             tstop = tstart + Quantity(livetime)
 
-        gti = GTI.create([tstart], [tstop], reference_time=reference_time)
+        if not isinstance(tstart, Time):
```

Comment:
Note that this could be moved to `GTI.create` to make it more general.

Reply:
Good idea!

Reply:
done